### PR TITLE
[Parse] Don't evaluate IfConfig condtion inside inactive block

### PR DESCRIPTION
**Pull request by rintaro**
```swift
#if false
#if canImport(SomeModule) // < This should not be evaluated
#endif
#endif
```

rdar://problem/50903021

### CODE REVIEW

**Comment by rintaro** [comment]
@swift-ci Please smoke test

File: test/Parse/ConditionalCompilation/canimport_in_inactive.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// RUN: mkdir -p %t/Modules/SomeModule.swiftmodule
+// RUN: echo 'DUMMY' > %t/Modules/SomeModule.swiftmodule/i386.swiftmodule
+// RUN: not %target-swift-frontend -typecheck -I %t/Modules %s 2>&1 | %FileCheck %s
+
+// REQUIRES: CPU=x86_64
+
+// Testing 'canImport()' not emitting error in inactive #if .. #endif blocks.
+
+#if false
+#if canImport(SomeModule) // Ok
```
Comment (rintaro) [inline test/Parse/ConditionalCompilation/canimport_in_inactive.swift]: Add `CHECK-NOT`

File: test/Parse/ConditionalCompilation/canimport_in_inactive.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#endif
+#endif
+
+#if true
+#else
+#if canImport(SomeModule) // Ok
```
Comment (rintaro) [inline test/Parse/ConditionalCompilation/canimport_in_inactive.swift]: Ditto

**Review by jrose-apple — APPROVED** [review]
Thank you!

**Comment by rintaro** [comment]
@swift-ci Please test

**Pull request merged**